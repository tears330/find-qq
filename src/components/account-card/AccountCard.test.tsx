import { render } from '@testing-library/react';
import { AccountCard } from '.';
describe('AccountCard', () => {
  it('render card element', () => {
    const { baseElement, rerender } = render(
      <AccountCard
        account={{
          code: 1,
          name: '木小末',
          qlogo: 'https://q2.qlogo.cn/headimg_dl?spec=100&dst_uin=876192777',
          qq: '876192777',
        }}
      />,
    );
    expect(baseElement).toMatchSnapshot();

    rerender(
      <AccountCard
        account={{
          code: 1,
          name: '木小末',
          qlogo: 'https://q2.qlogo.cn/headimg_dl?spec=100&dst_uin=876192777',
          qq: '7777777',
        }}
      />,
    );

    expect(baseElement).toMatchSnapshot();
  });
});
