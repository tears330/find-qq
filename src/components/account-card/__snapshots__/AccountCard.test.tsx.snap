// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountCard render card element 1`] = `
<body>
  <div>
    <div
      class="com-account-card"
    >
      <img
        alt="876192777"
        src="https://q2.qlogo.cn/headimg_dl?spec=100&dst_uin=876192777"
      />
      <div
        class="com-account-card-info"
      >
        <div
          class="com-account-card-info-name"
        >
          木小末
        </div>
        <div
          class="com-account-card-info-qq"
        >
          876192777
        </div>
      </div>
    </div>
  </div>
</body>
`;

exports[`AccountCard render card element 2`] = `
<body>
  <div>
    <div
      class="com-account-card"
    >
      <img
        alt="7777777"
        src="https://q2.qlogo.cn/headimg_dl?spec=100&dst_uin=876192777"
      />
      <div
        class="com-account-card-info"
      >
        <div
          class="com-account-card-info-name"
        >
          木小末
        </div>
        <div
          class="com-account-card-info-qq"
        >
          7777777
        </div>
      </div>
    </div>
  </div>
</body>
`;
